docs/management/admin/endpoint-protection-rules.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[endpoint-protection-rules]]
+= Endpoint protection rules
+
+Endpoint protection rules are prebuilt rules designed to help you manage and respond to alerts generated by {elastic-endpoint}, the installed component that performs {elastic-defend}'s threat monitoring and prevention. These rules include the <<endpoint-security, {elastic-defend}>> rule as well as additional detection and prevention rules for different {elastic-defend} protection features.
+
+NOTE: To receive {elastic-endpoint} alerts, you must install the Endpoint agent on your hosts (see <<install-endpoint>>).
+
+[discrete]
+== {elastic-defend} rule
+
+The {elastic-defend} rule automatically creates an alert from all incoming {elastic-endpoint} alerts. When this rule is enabled, the following Endpoint events are displayed as detection alerts:
+
+** Malware Prevention Alert
+** Malware Detection Alert
+
+NOTE: When you load the prebuilt rules, this is the only rule that is enabled by default.
+
+[discrete]
+== Feature-specific protection rules
+
+The following endpoint protection rules give you more granular control over how you handle the generated alerts. These rules are tailored for each of {elastic-defend}'s endpoint protection features—malware, ransomware, memory threats, and malicious behavior. Enabling these rules allows you to configure more specific actions based on the protection feature and whether the malicious activity was prevented or detected.
+
+* Behavior - Detected - Elastic Defend
+* Behavior - Prevented - Endpoint Defend
+* Malicious File - Detected - Elastic Defend
+* Malicious File - Prevented - Elastic Defend
+* Memory Signature - Detected - Elastic Defend
+* Memory Signature - Prevented - Elastic Defend
+* Ransomware - Detected - Elastic Defend
+* Ransomware - Prevented - Elastic Defend
+
+To use these rules, you need to manually enable them on the **Rules** page of the {security-app}. Follow the instructions for <<load-prebuilt-rules,installing and enabling Elastic prebuilt rules>>.
+
+[discrete]
+== Exception handling
```

Comment:
A more descriptive header could help with findability when users search our docs or Google them. (Same suggestion for [this title](https://github.com/elastic/security-docs/pull/6100/files#diff-2660cdd6a483bf299687cd255fea85908e3e47c4e2aad201cb73f4c193aaec29R37) in the Serverless docs.)

```suggestion
== Endpoint Security exception handling
```